### PR TITLE
Add skeleton-plus to your public CDN

### PR DESCRIPTION
**Pull request by smartclash**
Request to add this awesome library to your public CDN for public use

### CODE REVIEW

**Comment by smartclash** [comment]
@oltmannsdaniel Yuhu, Congrats to you Daniel. You've successfully submitted your project to the world's fastest and best CDN. Give me treat. Sweets ? :wink:

**Comment by megawac** [comment]
:tada:

**Comment by smartclash** [comment]
@megawac Hey, thanks to you too. Can I know why I am unable to search for the project or access the project manually?

When I search, I don't get the project. And when I try to access them manually, I find that the server responds with `File not found`. Is there any problem from my side?

File: files/skeleton-plus/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (megawac) [inline files/skeleton-plus/package.json]: whoops, this should be update.json

**Comment by megawac** [comment]
@smartclash there was an issue with the PR I missed. Thanks for commenting

**Comment by megawac** [comment]
I renamed the file, it will take up to 30 mins to take effect. Let me know
if its still not up by then

**Comment by smartclash** [comment]
@megawac Thanks a lot. It works :)

**Pull request merged**